aspnetcore/fundamentals/minimal-apis.md

```diff
@@ -1465,13 +1465,17 @@ The rules for determining a binding source from a parameter:
     1. Header: [`[FromHeader]`](xref:Microsoft.AspNetCore.Mvc.FromHeaderAttribute)
     1. Body: [`[FromBody]`](xref:Microsoft.AspNetCore.Mvc.FromBodyAttribute)
     1. Service: [`[FromServices]`](xref:Microsoft.AspNetCore.Mvc.FromServicesAttribute)
-    1. Parameter values: [`[AsParameters]](<xref:Microsoft.AspNetCore.Http.AsParametersAttribute)
+    1. Parameter values: [`[AsParameters]`](<xref:Microsoft.AspNetCore.Http.AsParametersAttribute)
 1. Special types
-    1. `HttpContext`
-    1. `HttpRequest`
-    1. `HttpResponse`
-    1. `ClaimsPrincipal`
-    1. `CancellationToken`
+    1. [`HttpContext`](xref:Microsoft.AspNetCore.Http.HttpContext)
+    1. [`HttpRequest`](xref:Microsoft.AspNetCore.Http.HttpContext.Request%2A)
+    1. [`HttpResponse`](xref:Microsoft.AspNetCore.Http.HttpContext.Response%2A)
+    1. [`ClaimsPrincipal`](xref:Microsoft.AspNetCore.Http.HttpContext.User%2A)
+    1. [`CancellationToken`](xref:Micrsoft.AspNetCore.Http.HttpContext.RequestAborted%2A)
+    1. [`IFormFileCollection`](xref:Microsoft.AspNetCore.Http.IFormFileCollection.Files%2A)
```

Comment:
```suggestion
    1. [`IFormFileCollection`](xref:Microsoft.AspNetCore.Http.IFormFileCollection)
```

Reply:
I meant for this to be `IFormCollection.Files`.

```suggestion
    1. [`IFormFileCollection`](xref:Microsoft.AspNetCore.Http.IFormCollection.Files)
```

Why do so many of the other property links in the repo use `%2A`?

Reply:
> Why do so many of the other property links in the repo use `%2A`?

@serpent5  wrote [the tool we use](https://rs-xrefs.azurewebsites.net/) , he  or @guardrex  might know. I just hit the copy button.

Reply:
Methods with overloads ... the overloads can't be linked individually and are represented by  an encoded *.

Reply:
Here's an example that shows the difference:

- `Microsoft.AspNetCore.Mvc.ControllerBase.Ok%2A`:
  - https://learn.microsoft.com/dotnet/api/microsoft.aspnetcore.mvc.controllerbase.ok
- `Microsoft.AspNetCore.Mvc.ControllerBase.Ok`:
  - https://learn.microsoft.com/dotnet/api/microsoft.aspnetcore.mvc.controllerbase.ok#microsoft-aspnetcore-mvc-controllerbase-ok